js/pricing.js

```diff
@@ -28,6 +58,67 @@ $(
                 disabledElement.attr('disabled', true);
                 disabledElement.addClass('disabled');
             }
+        };
+
+        /**
+         * Submit order handler.
+         */
+        const submitOrder = () => {
+            const dataProcessingConsent = $confirmPersonalInformation.prop("checked");
+            const supportAgreementConsent = $confirmDevelopmentAgreement.prop("checked");
+            const data = JSON.stringify({
+                "dataProcessingConsent": dataProcessingConsent,
+                "supportAgreementConsent": supportAgreementConsent
+            });
+
+            const transactionUrl = apiUrl + registerTransactionPath;
+            sendPaymentTransaction (transactionUrl, data);
+        };
+
+        /**
+         * Sends the transaction data and returns the transaction ID. If the request is successful,
+         * redirects to the Payment screen.
+         *
+         * @param {String} transactionUrl - the API URL
+         * @param {Object} data - the transaction data
+         */
+        const sendPaymentTransaction = (transactionUrl, data) => {
+            showRedirect(false);
+            $.ajax(transactionUrl, {
+                type: 'POST',
+                data: data,
+                contentType: 'application/json',
+                success: (data) => {
+                    const obj = JSON.parse(data);
+                    const paymentUrl = orderUrl + '&CUSTOMERID=' + obj.id;
+                    window.location = paymentUrl;
```

Comment:
From the UX perspective, should we notify the user that she'll be redirected to the payment processor website upon clicking on the button?

@JuliaEvseeva, @oleksandra-kovalenko, what do you think?

Reply:
We have this notification. Is is shown by `showRedirect(false);` function. 
![image](https://user-images.githubusercontent.com/17984255/90252567-657e2980-de48-11ea-9698-8678af42e3b1.png)

Reply:
How obvious. Let's just have 2 separate functions. A first one to just show the redirect splash screen. And the second one to show an error notification.

Reply:
Let's discuss it vocally?